KC Anchor (Update)

ThatOneGoblin25 opened the pull request:
## About the PR
- reworked atmos
- reworked pipes
- reworked wiring (SMES are looped to avoid AME flickering)
- reworked med, no more cryo and chem
- removed medical doctor locker
- removed nanomed
- removed booze & soda dispenser
- removed botany & kitchen
- removed vendors
- replaced blastdoors in bridge with shutters
- replaced chief engineer locker with engineer locker
- replaced T3 gunsafe with empty captain safe
- added two empty gun racks
- added boxing ring
- added disposal units around the ship
- added chaplain EVA and belt
- added janitor closet with disposals
- added television
- added two more arcades
- added cryosleep unit to morgue
- added surgery bed and body scanner (futureproofing:tm:)
- added telecomms for expedition
- decals and minor stuff
- shipyard price lowered

## Why / Balance
Good ship that needs a good update to return to the shipyard.

## Media
![anchor_2](https://github.com/user-attachments/assets/9e6fae39-ac44-48f3-a008-21a67d7ea8b8)

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**
:cl:
- add: KC Anchor returns to the expedition shipyard with a rework.

ThatOneGoblin25 commented:
Hey @Kesiath, letting you know there's some proposed changes to your ship.

dvir001 commented:
![image](https://github.com/user-attachments/assets/5daf0432-192b-4ab0-806b-fe8970b1cafa)
No medical beds, no medical kits, a single **wall locker** and that's it.

![image](https://github.com/user-attachments/assets/817fd62f-b218-470b-b666-5fe0f7bfbc28)
They will always spawn looking weird as spawners

![image](https://github.com/user-attachments/assets/1f600c6b-a80d-4855-acef-826725f305df)
No

ThatOneGoblin25 commented:
> No medical beds, no medical kits, a single wall locker and that's it.

Not even medical beds? Damn. As for the single wall locker it has the same fill as the medicine locker that's there.

> They will always spawn looking weird as spawners

![image](https://github.com/user-attachments/assets/decc338e-2c46-4180-8693-56177cbd3b72)
I personally think they look better than they do sideways because you get to see the screen.

> No

Missed that one.

ThatOneGoblin25 commented:
![Screenshot 2024-07-16 211750](https://github.com/user-attachments/assets/ff61cebb-0469-4285-a75c-09e0d04a8d92)

ThatOneGoblin25 commented:
This is fine.
![anchor_2](https://github.com/user-attachments/assets/cd31fde0-4059-466f-b5e4-2e021a672538)

dvir001 commented:
![image](https://github.com/user-attachments/assets/d1d8e299-8e80-4f6d-ab15-df4234dad8eb)

We avoid putting none table machines in tables since it cannot be done normally, need to be moved.

Other than that I don't see anything else that might be an issue from the pictures, leaving this to another maintainer review and merge.

ThatOneGoblin25 commented:
> We avoid putting none table machines in tables since it cannot be done normally, need to be moved.

Alright.

> Fix grid atmos again?

The tiles are like that intentionally, inspired by cluster station.

ThatOneGoblin25 commented:
![Screenshot 2024-07-18 233237](https://github.com/user-attachments/assets/88a71b9f-45c2-426f-baf4-213b34d08f18)

ErhardSteinhauer reviewed (changes requested):
Shuttle misses cargo palettes which makes it impossible to get rewards from expeditions.
Air scrubber in "sauna" syphons out water vapor.
The material reclaimer setup in custodian room is odd: it throws reclaimed usable materials into space.

ThatOneGoblin25 commented:
> Shuttle misses cargo palettes which makes it impossible to get rewards from expeditions. 

![Screenshot 2024-07-22 170504](https://github.com/user-attachments/assets/a4f8baee-fa71-4ad7-bb7f-4a89a443615f)
Ah so that's what there for, gotcha.

Air scrubber in "sauna" syphons out water vapor.

![Screenshot 2024-07-22 170217](https://github.com/user-attachments/assets/16867f17-91ee-484c-b08a-f14a82724e58)
It's now just its own loop to simplify it. There's a portable scrubber if theres a ammonia buildup ever

The material reclaimer setup in custodian room is odd: it throws reclaimed usable materials into space.

![Screenshot 2024-07-22 170445](https://github.com/user-attachments/assets/2e7308d8-8fe9-4aea-9c61-bbeaddb6e37a)
Should be fine now, can still sort out the junk not reclaimed

ThatOneGoblin25 commented:
![anchor3](https://github.com/user-attachments/assets/84b367c8-db16-452d-a4a7-f34f8d526dd5)
MapRender of final ver

K4rlox commented:
![image](https://github.com/user-attachments/assets/ba951254-617b-4b4a-93d8-98ae7da276a1)
way too many removals
theres no reason to remove any of these, you are just making it so its not even worth buying this ship anymore
whats the point of buying this ship after you removed all the roleplay value?

previously people had the reason to hire a bartender and a cook for roleplay, if this PR gets merged, they no longer need to

you replaced entirety of kitchen with a useless boxing room that adds nothing expect of reducing ship value and roleplay value


additionally, KC anchor is previously known as the only ship who has a dedicated room for a chaplain, not anymore


these changes feel like it was made to remove all roleplay value and discourage players from ever buying the ship

ThatOneGoblin25 commented:
I was told for any rework merge to be accepted I had to completely remove all vendors, kitchen and botany. 

> additionally, KC anchor is previously known as the only ship who has a dedicated room for a chaplain, not anymore

I didn't actually know this, I'll revisit adding more for chaplain

> you replaced entirety of kitchen with a useless boxing room that adds nothing expect of reducing ship value and roleplay value

Boxing ring felt like a cute rp idea for hard-headed mercs to sort out disputes for reward claiming. I also thought it would be a nice addition since the dedicated boxing ring ship, Knuckleverse was recently removed.

ErhardSteinhauer commented on Resources/Prototypes/_NF/Shipyard/Expedition/anchor.yml:
```diff
@@ -1,28 +1,28 @@
+  id: Anchor
+  name: KC Anchor
+  description: A large luxury cruiser capable of long ranged travel acrossed the sector, expedition capable.
+  price: 110100
```
Make sure that the markup is 30%

jimmy12or replied:
For what? No one is going to buy it at a high price anymore.

ThatOneGoblin25 replied:
Increased price by 7k for 30% markup. Grid sells for slightly less than 90k ingame.

ErhardSteinhauer reviewed (approved):
Looks good to me

arimah reviewed (changes requested):
A tiny number of tiny things:

![image](https://github.com/user-attachments/assets/d8f1084b-3048-43af-a75a-dd8bc3f74ee1)
This medical locker should probably not be filled.

![image](https://github.com/user-attachments/assets/bdc881dc-15bf-4a63-b2e4-7d5e64350daa)
The ship doesn't need _this many_ cargo pallets... You can probably reduce it to 4 or even 2.

![image](https://github.com/user-attachments/assets/e3bd4ba9-6c04-4560-a101-f89f76735aba)
Random arcade spawner has a chance of spawning a filled space villain arcade game. You can win plushies from it. I'd suggest replacing the spawners with an empty space villain arcade and an NT block game (facing west).

---

Beyond that, I can't see any technical issues with the ship. With so much being removed, unfortunately it feels like a big, dead space that doesn't really offer anything. I'm not sure who would really pick the Anchor, but if it's fine to have a mostly purposeless huge ship, then alright.

ThatOneGoblin25 commented:
> This medical locker should probably not be filled.

Dvir informed me I was allowed to have one filled medicine cabinet on the ship. I think it's fine to have a tiny amount of meds of a expo ship this large.

> The ship doesn't need this many cargo pallets... You can probably reduce it to 4 or even 2.

I'll reduce it to 4 in the corner

> Random arcade spawner has a chance of spawning a filled space villain arcade game. You can win plushies from it. I'd suggest replacing the spawners with an empty space villain arcade and an NT block game (facing west).

Alright.

> Beyond that, I can't see any technical issues with the ship. With so much being removed, unfortunately it feels like a big, dead space that doesn't really offer anything. I'm not sure who would really pick the Anchor, but if it's fine to have a mostly purposeless huge ship, then alright.

Yes unfortunately having to remove botany, kitchen and medical makes this ship quite a bit lackluster, it was a requirement for any approval from Dvir for it to lose it's multirole. However, for its size it's relatively cheap and if any crews want to set up their own space for stuff they have a plethora of options. I considered adding a salvage bay but I didn't really feel like it fit. If you had any suggestions to give it more purpose I'm welcome to suggestions, I just felt like this ship was too iconic to just leave removed.

arimah commented:
> Dvir informed me I was allowed to have one filled medicine cabinet on the ship. I think it's fine to have a tiny amount of meds of a expo ship this large.

Fair enough, then. If Dvir said you can have meds, you can have meds. :)

jimmy12or commented:
>Random arcade spawner has a chance of spawning a filled space villain arcade game. You can win plushies from it. I'd suggest replacing the spawners with an empty space villain arcade and an NT block game (facing west).

Just asking so I know, what's wrong with being able to win plushies from an arcade machine?

arimah commented:
> Just asking so I know, what's wrong with being able to win plushies from an arcade machine?

I must issue a correction: I forgot that these machines on Frontier give you prize tickets that you can cash in at Crazy Casey's Casino and honestly, thinking more about it, it doesn't really matter. It's fine, I'm sorry. The south-facing thing is more troubling. It looks super weird!

ThatOneGoblin25 commented:
Added a Chaplain EVA locker and belt 
<img width="328" alt="Screenshot 2024-08-13 at 4 06 55 PM" src="https://github.com/user-attachments/assets/3b52bcda-db18-453d-b929-e045aa441f22">

Replaced loose AME fuel with locker
<img width="484" alt="Screenshot 2024-08-13 at 4 19 39 PM" src="https://github.com/user-attachments/assets/3b4b8439-90eb-4b13-ba99-ab06d6de9244">

Reduced cargo pallets to four, rotated arcade machines and other very minor decal things

arimah reviewed (approved):
Seems to work fine, can't find any obvious oopsies. Good work fixing this ship up!
